ViewGenerator/tools/webpack/Rules/Files.ts

```diff
@@ -1,85 +1,91 @@
-﻿import { basename } from "path";
-import { RuleSetRule } from "webpack";
+﻿import {basename} from "path";
```

Comment:
Actually, this file is only used for managing imports of files like images, GIFs, etc 😅

Reply:
![image](https://github.com/OutSystems/ReactView/assets/13751333/b4857059-ff67-4bc8-876b-6283c8598ad1)

Reply:
We should just update here, or use an "ignore" rule provided by webpack

Reply:
done :)